example.js

```diff
@@ -121,6 +122,9 @@ shippo.customsdeclaration.create({
 	console.log("rates : %s", JSON.stringify(rates, null, 4));
 	// Get the first rate in the rates results for demo purposes.
 	rate = rates.results[0];
+	if (rates.results[0]) {
+		carrier_account = rates.results[0].carrier_account;
```

Comment:
is there any chance `rates.results[0].carrier_account` could be null or undefined?

Reply:
There should always be a carrier account for each shipment's rate